Initial Review.

jitesh-spryiq opened the pull request:
#### Descriptions:
_Add PR descriptions here_

#### Fix:
_Add fix/changes here_

jitesh-spryiq committed:
Initial Review.

cs-shivaji-kolape commented:
@jitesh-spryiq ,PR should not be empty. Please provide detailed descriptions, including:

1. Connector API Doc link
2. Whether actions were tested during setup or not
3. Whether unit testing was done or not

jitesh-spryiq committed:
Addressing review comments:
1. Modified the Connector API Doc link.
2. Tested all actions.
3. Completed unit testing.
